package/MDAnalysis/coordinates/base.py

```diff
@@ -137,6 +140,28 @@
 from . import core
 from .. import NoDataError
 
+from ..auxiliary.base import AuxReader
+from ..auxiliary.core import get_auxreader_for
+
+class Namespace(object):
```

Comment:
yes please. We don't know where it will be used in the future it is a very general concept that might prove more useful in the future.